addons/account/models/account_move.py

```diff
@@ -2085,6 +2085,19 @@ def _autopost_draft_entries(self):
         ])
         records.post()
 
+    # FIXME: Clarify me and change me in master
+    @api.multi
+    def action_duplicate(self):
+        self.ensure_one()
+        action = self.env.ref('account.action_move_journal_line').read()[0]
+        action['target'] = 'inline'
```

Comment:
```diff
-    action['target'] = 'inline'
```
this line was the result of a wong merge. It causes the breadcrumb and the top bar to go haywaire